packages/edge-gateway/src/utils/cid.js

```diff
@@ -32,3 +37,32 @@ export function normalizeCid(cid) {
   const c = CID.parse(cid)
   return c.toV1().toString()
 }
+
+/**
+ * Parse CID v1 and return normalized b32 v1
+ *
+ * @param {string} cid
+ */
+async function normalizeCidV1(cid) {
```

Comment:
Created new one as per CID spec when cidv0 we don't have a multibasePrefix. normalizeCidV1 always comes from this path as function is not exported and is used on subdomain